CMake/cdat_modules_extra/cdat_common_environment.cmake.in

```diff
@@ -1,5 +1,5 @@
 message("[INFO] ADDITIONAL CFLAGS ${ADDITIONAL_CFLAGS}")
-set(ENV{PATH} "@cdat_EXTERNALS@/bin:$ENV{PATH}")
+set(ENV{PATH} "@SB_BIN_DIR@:@cdat_EXTERNALS@/bin:$ENV{PATH}")
```

Comment:
Correct me if I am wrong but I don't think SB_BIN_DIR is common for all python or non-python packages. I prefer we do not stick it here.